apps/web-giddh/src/app/models/defaultMenus.ts

```diff
@@ -107,37 +110,34 @@ export let HIDE_NAVIGATION_BAR_FOR_LG_ROUTES = ['journal-voucher', 'inventory',
     'ledger'];
 export let DEFAULT_MENUS_ORIGINAL: IUlist[] = [
     {
-        type: 'MENU', name: 'Customer', uniqueName: '/pages/contact/customer', additional: { tab: 'customer', tabIndex: 0 }, isRemoved: false, pIndex: 3
+        type: 'MENU', name: 'Customer', uniqueName: '/pages/contact/customer', additional: { tab: 'customer', tabIndex: 0 }, isRemoved: false, pIndex: 3, hasTabs: true
     },
     {
-        type: 'MENU', name: 'GST', uniqueName: '/pages/gstfiling', isRemoved: false, pIndex: 5
+        type: 'MENU', name: 'GST', uniqueName: '/pages/gstfiling', isRemoved: false, pIndex: 5, hasTabs: false
     },
     {
-        type: 'MENU', name: 'Vat Report', uniqueName: '/pages/vat-report', isRemoved: false, pIndex: 5
+        type: 'MENU', name: 'Vat Report', uniqueName: '/pages/vat-report', isRemoved: false, pIndex: 5, hasTabs: false
     },
     {
-        type: 'MENU', name: 'Import', uniqueName: '/pages/import', isRemoved: false, pIndex: 10
+        type: 'MENU', name: 'Import', uniqueName: '/pages/import', isRemoved: false, pIndex: 10, hasTabs: false
     },
     {
-        type: 'MENU', name: 'Inventory', uniqueName: '/pages/inventory', isRemoved: false, pIndex: 8
+        type: 'MENU', name: 'Inventory', uniqueName: '/pages/inventory', isRemoved: false, pIndex: 8, hasTabs: true
     },
     {
-        type: 'MENU', name: 'Invoice', uniqueName: '/pages/invoice/preview/sales', isRemoved: false, pIndex: 2, additional: { tab: 'sales', tabIndex: 0 }
+        type: 'MENU', name: 'Invoice', uniqueName: '/pages/invoice/preview/sales', isRemoved: false, pIndex: 2, additional: { tab: 'sales', tabIndex: 0 }, hasTabs: true
     },
-    // {
-    //     type: 'MENU', name: 'Journal Voucher', uniqueName: '/pages/accounting-voucher', isRemoved: false, pIndex: 1
-    // },
     {
-        type: 'MENU', name: 'Journal Voucher *', uniqueName: '/pages/journal-voucher', isRemoved: false, pIndex: 1
+        type: 'MENU', name: 'Journal Voucher', uniqueName: '/pages/accounting-voucher', isRemoved: false, pIndex: 1
```

Comment:
Don't remove * and this is incorrect url

Reply:
Done